src/foam/u2/wizard/views/TipActionView.js

```diff
@@ -0,0 +1,48 @@
+foam.CLASS({
```

Comment:
Would this be better as a Border? I'm not saying that it would be. The toE() method could wrap it around what it gets back from SUPER().